op-e2e/actions/altda/altda_test.go

```diff
@@ -607,3 +643,64 @@ func TestAltDA_Finalization(gt *testing.T) {
 	// given 12s l1 time and 1s l2 time, l2 should be 12 * 3 = 36 blocks finalized
 	require.Equal(t, uint64(36), a.sequencer.SyncStatus().FinalizedL2.Number)
 }
+
+// This test tests ethDA -> altDA -> ethDA finalization behavior, simulating a temp altDA failure.
```

Comment:
nit:
altDA -> ethDA -> altDa, right?

Reply:
yes lol, thanks. Btw will be closing this, as its out of date and clearly won't be merged. Please use https://github.com/Layr-Labs/optimism/pull/23 going forward

Reply:
Fixed the typo in [11eb3d1](https://github.com/Layr-Labs/optimism/pull/23/commits/11eb3d1d9c017597acc32e67ee8ab6d97017c04a) btw